packages/forms-web-app/src/app.js

```diff
@@ -83,9 +84,8 @@ if (config.server.useSecureSessionCookie) {
 	app.set('trust proxy', 1); // trust first proxy
 }
 
+app.disable('x-powered-by');
```

Comment:
this is such a classic security recommendation. I find it funny that they always insist we add this even in open source code